scripts/run-all-tests.sh

```diff
@@ -0,0 +1,20 @@
+examples=(
+  "dex"
+  "oracle"
+  "erc20"
+  "state-rent"
+  "e2e"
+  "scheduler"
```

Comment:
Rename `deploy` to `test` in `examples-arbitrager` and `examples-uniswap`.

Maybe can merge `trade.ts` into `deploy.ts` in `examples-uniswap`, we can test it directly